internal/config/migrate/from_1_20.go

```diff
@@ -0,0 +1,18 @@
+package migrate
+
+import (
+	"github.com/cri-o/cri-o/pkg/config"
+	"github.com/sirupsen/logrus"
+)
+
+// migrateFrom1_20 migrates a config from the 1.20.x version
+func migrateFrom1_20(cfg *config.Config) error {
+	// Upgrade pause image
+	logrus.Infof("Checking for pause_image, which now should be k8s.gcr.io/pause:3.5 instead of k8s.gcr.io/pause:3.2")
+	if cfg.PauseImage == "k8s.gcr.io/pause:3.2" {
```

Comment:
Would this be better?
```suggestion
	if cfg.PauseImage != config.DefaultPauseImage {
```

Reply:
This would change the value to `k8s.gcr.io/pause:3.2`, which would change the logic there :)